tools/codegen/local.py

```diff
@@ -17,24 +15,10 @@
 # DON'T add a new entry here.
 
 class Locals(threading.local):
-    use_c10_dispatcher: Optional[UseC10Dispatcher] = None
-_locals = Locals()
```

Comment:
This looks like we can remove the whole Locals class, cc @ezyang did you want to use it for something else?

Reply:
Discussed this offline with @ezyang . Supporting dynamic scoping during codegen seems a useful feature, plus is not very much code.